screens/first/first.js

```diff
@@ -21,6 +20,8 @@ export default class FirstScreen extends Component {
   render() {
     let { selected }  = this.state;
 
+    const spinnerUri = 'https://www.companyshop.co.uk/media/1017/media-placeholder.jpg';
```

Comment:
it's better to store such values in JSON somewhere, eg config.json. But for homework it's ok